applications/rag/README.md

```diff
@@ -1,192 +1,204 @@
-# RAG-on-GKE Application
+# RAG on GKE
 
-**NOTE:** This solution is in beta. Please expect friction while using it.
+## What is RAG?
 
-This is a sample to deploy a RAG application on GKE. Retrieval Augmented Generation (RAG) is a popular approach for boosting the accuracy of LLM responses, particularly for domain specific or private data sets. The basic idea is to have a semantically searchable knowledge base (often using vector search), which is used to retrieve relevant snippets for a given prompt to provide additional context to the LLM. Augmenting the knowledge base with additional data is typically cheaper than fine tuning and is more scalable when incorporating current events and other rapidly changing data spaces.
+This is a sample to deploy a Retrieval Augmented Generation (RAG) application on GKE. RAG is a popular approach for boosting the accuracy of LLM responses, particularly for domain specific or private data sets.
 
-Architecture:
-1. A k8s service serving Hugging Face TGI inference using `mistral-7b`.
-2. Cloud SQL `pgvector` instance with vector embeddings generated from the input dataset.
-3. A k8s front end chat interface to interact with the inference server and the vector DB.
-4. Ray cluster runs job to populate vector DB.
-5. Jupyter notebook to read dataset and trigger ray job to populate the vector DB.
+RAG uses a semantically searchable knowledge base (like vector search) to retrieve relevant snippets for a given prompt to provide additional context to the LLM. Augmenting the knowledge base with additional data is typically cheaper than fine tuning and is more scalable when incorporating current events and other rapidly changing data spaces.
```

Comment:
Consider adding a link for more information on RAG.

Reply:
done